### PR TITLE
Small fix: add keyboard esc support for color picker

### PR DESCRIPTION
**Pull request by dvoytenko**

### CODE REVIEW

File: assets/src/edit-story/components/colorPicker/index.js [hunk 1]
```diff
@@ -92,6 +93,9 @@ function ColorPicker({ color, hasGradient, hasOpacity, onChange, onClose }) {
   const previousFocus = useRef(document.activeElement);
 
   useFocusOut(containerRef, onClose);
+  useKeyDownEffect(containerRef, { key: 'esc', editable: true }, onClose, [
+    onClose,
+  ]);
```
Comment (barklund) [inline assets/src/edit-story/components/colorPicker/index.js:98]: [<kbd>esc</kbd> support is coming](https://github.com/google/web-stories-wp/pull/497/files#diff-9eae147e026beb102527eb1a924fc5c7R122) in #497, FYI.

Comment (barklund) [reply assets/src/edit-story/components/colorPicker/index.js:98]: And the lack of `{ editable: true }` in that line in 497 is actually intentional. If you're editing either hex or opacity, you can press <kbd>esc</kbd> to just exit editing this field only, but not close the entire color picker. Pressing <kbd>esc</kbd> again will then close the color picker.

Comment (swissspidy) [reply assets/src/edit-story/components/colorPicker/index.js:98]: > Pressing <kbd>ESC</kbd> again will then close the color picker.

@barklund That doesn't seem to always work, e.g. when changing the page background color. And when editing text color, it not only closes the the color picker, but actually unselects the text element.

Comment (barklund) [reply assets/src/edit-story/components/colorPicker/index.js:98]: Hm, those are good points. We will need to check the event flow better for this to work consistently.

Comment (dvoytenko) [reply assets/src/edit-story/components/colorPicker/index.js:98]: Ok. So, I can close this then, right?

Comment (swissspidy) [reply assets/src/edit-story/components/colorPicker/index.js:98]: I think so, yes.

@barklund can you make sure there's an issue for the things I mentioned?

**Pull request closed without merging**